[License] Keep the Apache License for not changed files

chaoyli opened the pull request:
Many files come from Apache Doris and have not been changed.
According to the License rule, the files should keep the Apache License.

## What type of PR is this：
- [ ] bug
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：
<!--
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*
-->
Fixes #

## Problem Summary(Required) ：
<!-- (Please describe the changes you have made. In which scenarios will this bug be triggered and what measures have you taken to fix the bug?) -->

chaoyli committed:
[License] Keep the Apache License for not changed files

Many files come from Apache Doris and have not been changed.
According to the License rule, the files should keep the Apache License.

imay commented on fe/fe-core/src/main/java/com/starrocks/analysis/SchemaTableType.java:
```diff
@@ -1,7 +1,3 @@
-// This file is made available under Elastic License 2.0.
```
This should keep the this header

chaoyli committed:
[License] Keep the Apache License for not changed files

Many files come from Apache Doris and have not been changed.
According to the License rule, the files should keep the Apache License.

wanpengfei-git commented:
#### [FE PR Coverage check]


:heart_eyes: **pass** : 0 / 0 (0%)
